app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):

```

Comment:
Inherit `Elf` from the `ABC` as well to indicate that this class is abstract since it does not implement all abstract methods from the `Player` class